sockeye/inference.py

```diff
@@ -630,6 +635,62 @@ def make_input(sentence_id: int, sentence: str) -> TranslatorInput:
         tokens = list(data_io.get_tokens(sentence))
         return TranslatorInput(id=sentence_id, sentence=sentence.rstrip(), tokens=tokens)
 
+    def batch_translate(self, trans_inputs: List[TranslatorInput]) -> List[TranslatorOutput]:
```

Comment:
since you are not calling `translate()` anymore, why not rename batch_translate to translate?

Reply:
Fixed